### ISSUE TITLE
java.lang.NullPointerException

### ISSUE THREAD

**Issue by h266579110** [zhenghanlee/LargeRaids#4]
Not sure how to happen this, just saw from console.
PaperMC 1.18.2-267

```
[19:50:04 ERROR]: Could not pass event SpawnerSpawnEvent to LargeRaids v1.10.3
java.lang.NullPointerException: Cannot invoke "net.minecraft.world.entity.EntityInsentient.getBukkitEntity()" because the return value of "net.minecraft.world.entity.monster.EntityVex.q()" is null
        at com.solarrabbit.largeraids.v1_18_R2.nms.CraftVexWrapper.getOwner(CraftVexWrapper.java:18) ~[LargeRaids.jar:?]
        at com.solarrabbit.largeraids.raid.mob.EventBomber.onSummonVex(EventBomber.java:80) ~[LargeRaids.jar:?]
        at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:37) ~[paper-api-1.18.2-R0.1-SNAPSHOT.jar:?]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[paper-api-1.18.2-R0.1-SNAPSHOT.jar:git-Paper-267]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[paper-api-1.18.2-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:629) ~[paper-api-1.18.2-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_18_R2.event.CraftEventFactory.callSpawnerSpawnEvent(CraftEventFactory.java:1688) ~[paper-1.18.2.jar:git-Paper-267]
        at net.minecraft.world.level.BaseSpawner.serverTick(BaseSpawner.java:186) ~[?:?]
        at net.minecraft.world.level.block.entity.SpawnerBlockEntity.serverTick(SpawnerBlockEntity.java:54) ~[?:?]
        at net.minecraft.world.level.chunk.LevelChunk$BoundTickingBlockEntity.tick(LevelChunk.java:1207) ~[?:?]
        at net.minecraft.world.level.chunk.LevelChunk$RebindableTickingBlockEntityWrapper.tick(LevelChunk.java:1159) ~[?:?]
        at net.minecraft.world.level.Level.tickBlockEntities(Level.java:917) ~[?:?]
        at net.minecraft.server.level.ServerLevel.tick(ServerLevel.java:682) ~[?:?]
        at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1591) ~[paper-1.18.2.jar:git-Paper-267]
        at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:483) ~[paper-1.18.2.jar:git-Paper-267]
        at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1455) ~[paper-1.18.2.jar:git-Paper-267]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1225) ~[paper-1.18.2.jar:git-Paper-267]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:315) ~[paper-1.18.2.jar:git-Paper-267]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
[19:50:04 ERROR]: Could not pass event CreatureSpawnEvent to LargeRaids v1.10.3
java.lang.NullPointerException: Cannot invoke "net.minecraft.world.entity.EntityInsentient.getBukkitEntity()" because the return value of "net.minecraft.world.entity.monster.EntityVex.q()" is null
        at com.solarrabbit.largeraids.v1_18_R2.nms.CraftVexWrapper.getOwner(CraftVexWrapper.java:18) ~[LargeRaids.jar:?]
        at com.solarrabbit.largeraids.raid.mob.EventBomber.onSummonVex(EventBomber.java:80) ~[LargeRaids.jar:?]
        at com.destroystokyo.paper.event.executor.MethodHandleEventExecutor.execute(MethodHandleEventExecutor.java:37) ~[paper-api-1.18.2-R0.1-SNAPSHOT.jar:?]
        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[paper-api-1.18.2-R0.1-SNAPSHOT.jar:git-Paper-267]
        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[paper-api-1.18.2-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:629) ~[paper-api-1.18.2-R0.1-SNAPSHOT.jar:?]
        at org.bukkit.craftbukkit.v1_18_R2.event.CraftEventFactory.callCreatureSpawnEvent(CraftEventFactory.java:739) ~[paper-1.18.2.jar:git-Paper-267]
        at org.bukkit.craftbukkit.v1_18_R2.event.CraftEventFactory.doEntityAddEventCalling(CraftEventFactory.java:644) ~[paper-1.18.2.jar:git-Paper-267]
        at net.minecraft.server.level.ServerLevel.addEntity(ServerLevel.java:1422) ~[?:?]
        at net.minecraft.server.level.ServerLevel.addFreshEntity(ServerLevel.java:1331) ~[?:?]
        at net.minecraft.world.level.ServerLevelAccessor.lambda$addFreshEntityWithPassengers$0(ServerLevelAccessor.java:16) ~[?:?]
        at java.util.stream.Streams$StreamBuilderImpl.forEachRemaining(Streams.java:411) ~[?:?]
        at java.util.stream.ReferencePipeline$Head.forEach(ReferencePipeline.java:762) ~[?:?]
        at net.minecraft.world.level.ServerLevelAccessor.addFreshEntityWithPassengers(ServerLevelAccessor.java:16) ~[?:?]
        at net.minecraft.server.level.ServerLevel.tryAddFreshEntityWithPassengers(ServerLevel.java:1445) ~[?:?]
        at net.minecraft.world.level.BaseSpawner.serverTick(BaseSpawner.java:198) ~[?:?]
        at net.minecraft.world.level.block.entity.SpawnerBlockEntity.serverTick(SpawnerBlockEntity.java:54) ~[?:?]
        at net.minecraft.world.level.chunk.LevelChunk$BoundTickingBlockEntity.tick(LevelChunk.java:1207) ~[?:?]
        at net.minecraft.world.level.chunk.LevelChunk$RebindableTickingBlockEntityWrapper.tick(LevelChunk.java:1159) ~[?:?]
        at net.minecraft.world.level.Level.tickBlockEntities(Level.java:917) ~[?:?]
        at net.minecraft.server.level.ServerLevel.tick(ServerLevel.java:682) ~[?:?]
        at net.minecraft.server.MinecraftServer.tickChildren(MinecraftServer.java:1591) ~[paper-1.18.2.jar:git-Paper-267]
        at net.minecraft.server.dedicated.DedicatedServer.tickChildren(DedicatedServer.java:483) ~[paper-1.18.2.jar:git-Paper-267]
        at net.minecraft.server.MinecraftServer.tickServer(MinecraftServer.java:1455) ~[paper-1.18.2.jar:git-Paper-267]
        at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1225) ~[paper-1.18.2.jar:git-Paper-267]
        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:315) ~[paper-1.18.2.jar:git-Paper-267]
        at java.lang.Thread.run(Thread.java:833) ~[?:?]
```

**Comment by solarrabbit99** [comment]
Thanks for the report! It happened because you have a vex spawner in your server and my plugin couldn't locate vex's owner (the evoker that summoned it). It shouldn't effect gameplay but I will get it fixed by the next update.

**Comment by solarrabbit99** [comment]
Should be fixed in v1.10.4. Update the thread if the problem is unresolved.
